src/components/app/create/CreateApp.tsx

```diff
@@ -332,20 +349,41 @@ export class AddNewApp extends Component<AddNewAppProps, AddNewAppState> {
                             Apps are NOT env specific and can be used to deploy to multiple environments.
                         </span>
                     )}
-                    {this.props.isJobView && (
-                        <>
-                            <span className="form__label">Description</span>
+                    <>
+                        <span className="form__label mt-16">Description</span>
+                        {this.state.showClusterDescription ? (
+                            <div className="h-auto">
+                                <ClusterDescription
+                                    isClusterTerminal={false}
+                                    isSuperAdmin={true}
+                                    appId={this.state.form.appId}
+                                    descriptionId={0}
+                                    initialDescriptionText={''}
```

Comment:
Can we avoid passing these empty props ?

Reply:
done

Reply:
Not yet done

Reply:
done now